### ISSUE TITLE
Conclusion of design session: two vocabularies

### ISSUE THREAD

**Issue by jgonggrijp** [UUDigitalHumanitieslab/edpop-collection-ontology#1]
@tijmenbaarda, @lukavdplas and I discussed the RDF vocabulary to be used for records and collections in EDPOP. We ended up with the conclusion that we need TWO vocabularies, which will be used together in the VRE but be definined independently:

1. A vocabulary that describes catalogs and records (in biographic and bibliographic flavors), the "record ontology". 
2. A vocabulary that describes collection membership and its interplay with annotations (building on top of the web annotation vocabulary), the "collection ontology".

The explorer will exclusively use the record ontology. The VRE will retrieve data in this format from the explorer and then plug them into the collection ontology. The VRE will likely hardcode the collection ontology, but its frontend might not need to make assumptions about the record ontology.

@tijmenbaarda will be writing the initial version of the record ontology while @lukavdplas will write the initial version of the collection ontology. The current repo, due to its name, should be used only for the latter vocabulary; @tijmenbaarda will be creating another repo for the record ontology.

Helpful links in the writing process:

- https://www.w3.org/TR/rdf11-primer/
- https://www.w3.org/TR/rdf-schema/
- https://www.w3.org/TR/owl2-primer/
- https://www.w3.org/TR/skos-primer/
- https://www.w3.org/TR/annotation-vocab/

I will be adding a picture of Luka's whiteboard drawing and some "sketch code" of what a vocabulary looks like in subsequent posts.

FYI @JeltevanBoheemen

**Comment by jgonggrijp** [comment]
Top left: collection ontology, bottom right: record ontology.

![CD755D45-3362-4E2F-882E-48D66C341110](https://github.com/UUDigitalHumanitieslab/edpop-collection-ontology/assets/1540185/2da6d343-6bad-4d87-9c01-d3dda86c49b3)

**Comment by jgonggrijp** [comment]
The "trash code" I wrote in Turtle to give a taste of what writing an RDF vocabulary looks like. This started under the assumption that we would be creating a single ontology, so this is a mixture of collections and records.

```turtle
@prefix edpop: <https://dhstatic.hum.uu.nl/edpop-collections/latest/>
@prefix rdf: <>
@prefix rdfs: <>
@prefix owl: <>
@prefix skos: <>
@prefix dcterms: <>
@prefix xsd: <>

subject predicate object.
subject predicate object1,
                  object2.
subject predicate1 object1;
        predicate2 object2.

edpop: a owl:Ontology.

edpop:Catalog a rdfs:Class;
              skos:prefLabel "Catalog"^en
              skos:prefLabel "Catalogus"^nl
              skos:description "External repository of bibliographical or biographical data".

edpop:Bibliography rdfs:subClassOf edpop:Catalog.
edpop:Biography rdfs:subClassOf edpop:Catalog.

edpop:Collection a rdfs:Class
                 owl:equivalentClass [
                 rdf:type owl:Restriction;
                 owl:onProperty dcterms:name;
                 owl:someValuesFrom xsd:string.
].
# bovenstaande is equivalent aan onderstaande
edpop:Collection a rdfs:Class
                 owl:equivalentClass _:1.

_:1              rdf:type owl:Restriction;
                 owl:onProperty dcterms:name;
                 owl:someValuesFrom xsd:string.

edpop:Record a rdfs:Class.

edpop:inCollection a rdf:Property;
                   rdfs:domain edpop:record;
                   rdfs:range edpop:Collection;
                   skos:prefLabel "member of collection";
                   skos:description "".

dcterms:name rdfs:domain edpop:Collection.

edpop:name rdfs:subPropertyOf dcterms:name.

# Below is not Turtle but SPARQL (and not part of writing an ontology).
construct {
          ?record ?p ?o.
} where {
  ?record a edpop:Record.
  ?record dcterms:name ?name.
}
```

**Comment by tijmenbaarda** [comment]
What license should we give the ontologies? Did we apply a license to our previous ontologies (e.g. read-it)?

I am following the Pizza tutorial of Protege and I see that they use CC-BY: http://protege.stanford.edu/ontologies/pizza/pizza.owl

**Comment by jgonggrijp** [comment]
CC-BY seems sensible to me. We could ask Jeroen and Andrea for their opinions.

**Comment by lukavdplas** [comment]
closed by #2
